randovania/games/dread/json_data/header.json

```diff
@@ -1199,7 +1199,7 @@
                 "extra": {}
             },
             "HighDanger": {
-                "long_name": "Highly Dangerous Logic",
+                "long_name": "Allow Highly Dangerous Logic",
```

Comment:
I'd prefer this stays how it was - `Enabled Highly Dangerous Logic` looks better in the db than `Enabled Allow Highly Dangerous Logic`

Reply:
ah, i wasnt aware it took it from here. i'll change it back